Validations/Validation_2022_w5_HCal

```diff
@@ -0,0 +1,66 @@
+# Email/HN link through which this validation is requested.
+# Put is None in case you don't have any link
+ValidationRequest       : https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/4583.html
+
+Title                   : JEC tag for Run3 
+
+Subsystem               : HCal
```

Comment:
Should this be JETMET instead?

Reply:
I see that JetCorrectionsRecord is categorized in HCAL
From this table in Run2 :
https://twiki.cern.ch/twiki/bin/viewauth/CMS/PdmVTriggerConditionValidation2018#Full_Validation

But could be changed if the JetMET category is an appropriate label. Should I?

Reply:
Yes please